CoreGraphUtilities/src/au/gov/asd/tac/constellation/graph/utilities/io/CopyGraphUtilities.java

```diff
@@ -122,20 +122,11 @@ public static void copyGraphToGraph(final GraphReadMethods original, final Graph
         final int[] linkTranslation = new int[original.getLinkCapacity()];
         for (int position = 0; position < original.getTransactionCount(); position++) {
             final int originalTransaction = original.getTransaction(position);
-            if (!copyAll) {
-                if (transactionSelected != Graph.NOT_FOUND) {
-                    if (!original.getBooleanValue(transactionSelected, originalTransaction)) {
-                        continue;
-                    }
-                }
-                if (vertexSelected != Graph.NOT_FOUND) {
-                    if (!original.getBooleanValue(vertexSelected, original.getTransactionSourceVertex(originalTransaction))) {
-                        continue;
-                    }
-                    if (!original.getBooleanValue(vertexSelected, original.getTransactionDestinationVertex(originalTransaction))) {
-                        continue;
-                    }
-                }
+            if (!copyAll && ((transactionSelected != Graph.NOT_FOUND && !original.getBooleanValue(transactionSelected, originalTransaction)) ||
```

Comment:
if you don't have a preference, can the `||` and `&&` be added to the start of the line rather than the end?

Reply:
Done